sdk/cosmos/azure_data_cosmos/src/models/mod.rs

```diff
@@ -39,7 +41,9 @@ where
 #[non_exhaustive]
 #[derive(Clone, Default, Debug, Deserialize)]
 pub struct QueryResults<T> {
-    #[serde(rename = "Documents")]
+    #[serde(alias = "Documents")]
+    #[serde(alias = "Databases")]
+    #[serde(alias = "DocumentCollections")]
```

Comment:
I'm a little concerned these could diverge after a time. [REST API guidelines](https://aka.ms/azapi/guidelines) dictate services never return a top-level array because it's inflexible. You're not doing that here, which is good; however, it implies because you can add fields in the future. So, could a Documents vs. Databases vs. DocumentCollections result ever add differing fields which would necessitate a different type or at least using an enum here instead?